Tests/test_bgzf.py

```diff
@@ -126,9 +124,13 @@ def check_by_line(self, old_file, new_file, old_gzip=False):
                 h = open(old_file, mode)
             old = h.read()
             if "b" in mode:
-                old = _as_bytes(old)
+                if isinstance(old, str):
+                    import codecs
+                    old = codecs.latin_1_encode(old)[0]
```

Comment:
@peterjc While we are looking at this and as @Andrey-Raspopov touched [here](https://github.com/biopython/biopython/pull/2445#issuecomment-569322879), I am actually wondering what is the rationale to use (and keep using) latin-1 encoding as opposed to utf-8?

There may be [some cases](https://github.com/biopython/biopython/pull/1547) where that makes sense as the default encoding. Also, historically it seems that latin-1 was at one point a competing Unicode encoding scheme. However, it seems that nowadays it is UTF-8 that is the de-facto unicode encoding standard. It is also [the default encoding that Python source files have](https://docs.python.org/3/howto/unicode.html#unicode-literals-in-python-source-code), sans an encoding comment.

Is there a reason (other than maybe backwards compatibility) that we keep using `latin-1`?

For reference, I also tried looking at our history and can see that the [last commit that introduced `latin-1` was made almost 9 years ago](https://github.com/biopython/biopython/commit/ffcdd7d51763ff15151913af525ab596edd2b4c6).

Reply:
I agree that UTF-8 makes more sense nowadays

Reply:
See also my comment above for `Bio/bgzf.py`.